internal/configs/version1/nginx-plus.tmpl

```diff
@@ -65,7 +65,7 @@ http {
 
     map $http_upgrade $connection_upgrade {
```

Comment:
Please configure a different map so that your change doesn't interfere with the implementation for the Ingress resource. To configure a different map, choose a different variable. For example:
```yaml
map $http_upgrade $vs_connection_upgrade {
```